src/main/java/delete_event_use_case/DeleteEventDataAccess.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Delete Event Data Access class.
+ * @author : Khizer Ahmad
+ * @created: November 19th 2022
+ * @since : 2022-11-28, Mon.
+ **/
+
+package delete_event_use_case;
+
+import data_access.DataAccessDay;
+import data_access.DataAccessEvent;
+import data_access.WeekDataAccess;
+
+import java.util.HashMap;
+import java.util.ArrayList;
+
+public class DeleteEventDataAccess extends WeekDataAccess implements DeleteEventDsGateway {
+
+    @Override
+    public boolean eventExists(int dayIndex, String eventTitle) {
+        DataAccessDay day = super.getDays().get(dayIndex);
```

Comment:
You can directly access the days instance of WeekDataAccess since it is a protected instance attribute. Hence, replace super.getDays() with days

Reply:
Done, thank you.